mlir/include/mlir/Dialect/OpenMP/OpenMPOps.td

```diff
@@ -187,8 +188,10 @@ def ParallelOp : OpenMP_Op<"parallel", [
              Variadic<AnyType>:$allocate_vars,
              Variadic<AnyType>:$allocators_vars,
              Variadic<OpenMP_PointerLikeType>:$reduction_vars,
+             Variadic<AnyType>:$private_vars,
              OptionalAttr<SymbolRefArrayAttr>:$reductions,
-             OptionalAttr<ProcBindKindAttr>:$proc_bind_val);
+             OptionalAttr<ProcBindKindAttr>:$proc_bind_val,
+             OptionalAttr<SymbolRefArrayAttr>:$private_inits);
```

Comment:
Does each element of `private_inits` correspond to a PrivateClauseOp?
If this is the case, I think that a name like `private_ops` or `privatizers` would make it clearer what this attributes array is supposed to contain. The `_inits` suffix makes me think it holds the symbols from which values the privatized vars will be initialized.

Reply:
Done